docs/index.md

```diff
@@ -0,0 +1,81 @@
+# MLBox 
+
+This is the MLBox 📦 Prototype. This is  still under construction, some parts probably don't work yet, or may have unexpected/inconsistent behaviours.
```

Comment:
IMHO I think we should start the first docs with correct contents. 
index.md with incorrect changes can be avoided for mkdocs PR.

Reply:
Addressed